honeybadgermpc/progs/mimc_jubjub_pkc.py

```diff
@@ -0,0 +1,77 @@
+import asyncio
+from random import randint
+from honeybadgermpc.field import GF
+from honeybadgermpc.mpc import PreProcessedElements, Subgroup
+from honeybadgermpc.elliptic_curve import Point, Jubjub
+from honeybadgermpc.progs.jubjub import share_mul
+from honeybadgermpc.progs.mimc import mimc_mpc, mimc_plain
+
+pp_elements = PreProcessedElements()
+field = GF(Subgroup.BLS12_381)
```

Comment:
hmm, I see why you need this though since the one method doesn't take in a context... but taking in a field argument would probably be better in that method. This could cause problems down the road otherwise if we add in more fields.

Reply:
I removed this line. Now in `mimc_encrypt` method I use `Jubjub.Field` instead, though the problem when more fields added still exists.